Mathlib/Topology/Algebra/Module/StrongTopology.lean

```diff
@@ -185,6 +192,61 @@ theorem hasBasis_nhds_zero [TopologicalSpace F] [TopologicalAddGroup F]
       { f : UniformConvergenceCLM σ F 𝔖 | ∀ x ∈ SV.1, f x ∈ SV.2 } :=
   hasBasis_nhds_zero_of_basis σ F 𝔖 h𝔖₁ h𝔖₂ (𝓝 0).basis_sets
 
+theorem nhds_zero_eq_of_basis [TopologicalSpace F] [TopologicalAddGroup F] (𝔖 : Set (Set E))
+    {ι : Type*} {p : ι → Prop} {b : ι → Set F} (h : (𝓝 0 : Filter F).HasBasis p b) :
+    𝓝 (0 : UniformConvergenceCLM σ F 𝔖) =
+      ⨅ (s : Set E) (_ : s ∈ 𝔖) (i : ι) (_ : p i),
+        𝓟 {f : UniformConvergenceCLM σ F 𝔖 | MapsTo f s (b i)} := by
+  letI : UniformSpace F := TopologicalAddGroup.toUniformSpace F
+  haveI : UniformAddGroup F := comm_topologicalAddGroup_is_uniform
+  rw [(embedding_coeFn σ F 𝔖).toInducing.nhds_eq_comap,
+    UniformOnFun.nhds_eq_of_basis _ _ h.uniformity_of_nhds_zero]
+  simp [MapsTo]
+
+theorem nhds_zero_eq [TopologicalSpace F] [TopologicalAddGroup F] (𝔖 : Set (Set E)) :
+    𝓝 (0 : UniformConvergenceCLM σ F 𝔖) =
+      ⨅ (s : Set E) (_ : s ∈ 𝔖) (t : Set F) (_ : t ∈ 𝓝 0),
+        𝓟 {f : UniformConvergenceCLM σ F 𝔖 | MapsTo f s t} :=
+  nhds_zero_eq_of_basis _ _ _ (𝓝 0).basis_sets
+
+variable {F} in
+theorem gen_mem_nhds_zero [TopologicalSpace F] [TopologicalAddGroup F]
```

Comment:
What's the meaning of `gen` here? If we don't have precedence for this name elsewhere, and you can't think of a better name, can you add a docstring expanding it?

Reply:
I guess, it's from something like "generator". I copied from [UniformOnFun.gen_mem_nhds](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/UniformSpace/UniformConvergenceTopology.html#UniformOnFun.gen_mem_nhds)

Reply:
I'm going to reformulate it using `Eventually` and rename it.